packages/tests/src/lib/index.ts

```diff
@@ -1,2 +1,294 @@
-export * from "./message_collector.js";
-export * from "./service_node.js";
+import { DecodedMessage } from "@waku/core";
+import {
+  DefaultPubsubTopic,
+  PubsubTopic,
+  ShardingParams
+} from "@waku/interfaces";
+import { Logger } from "@waku/utils";
+import { expect } from "chai";
+
+import { Args, MessageRpcQuery, MessageRpcResponse } from "../types";
+import { delay, makeLogFileName } from "../utils/index.js";
+
+import { MessageCollector } from "./message_collector.js";
+import { defaultArgs, ServiceNode } from "./service_node.js";
+
+export { ServiceNode, MessageCollector, defaultArgs };
+
+const log = new Logger("test:message-collector");
+
+/**
+ * This class is a wrapper over the ServiceNode & MessageCollector class
+ * that allows for the creation & handling of multiple ServiceNodes
+ */
+export class ServiceNodes {
```

Comment:
`ServiceNodes` and `ServiceNode` are a bit hard to differentiate. Maybe `ServiceNodeFleet` is better?

Reply:
sure!